src/status.rs

```diff
@@ -458,6 +464,52 @@ impl StatusBuilder {
         self
     }
 
+    pub fn spacefed(mut self, spacefed: Spacefed) -> Self {
+        self.spacefed = Some(spacefed);
+        self
+    }
+
+    pub fn add_event(mut self, event: Event) -> Self {
+        if let Some(ref mut events) = self.events {
+            events.push(event);
+        } else {
+            self.events = Some(vec![event]);
+        }
```

Comment:
I found that the combinators don't make for much nicer code, since you cannot use `map()` etc to push into the vector. If you want, I can change it to something like this:

```rust
pub fn add_radio_show(mut self, radio_show: RadioShow) -> Self {
    let mut shows = self.radio_show.unwrap_or_else(Vec::new);
    shows.push(radio_show);
    self.radio_show = Some(shows);
    self
}
```
I don't mind.